.github/workflows/build.yml

```
@@ -9,7 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Setup NuGet
-        uses: NuGet/setup-nuget@v1.0.5
+        uses: NuGet/setup-nuget@v2.0.0
```

```suggestion
        uses: NuGet/setup-nuget@v2
```

I believe this will work still.

---

lets find out!